test/hotspot/jtreg/runtime/modules/IgnoreModulePropertiesTest.java

```diff
@@ -58,21 +58,27 @@ public static void testProperty(String prop, String value) throws Exception {
     // For options of the form "option=value", check that an exception gets thrown for
     // the illegal value and then check that its corresponding property is handled
     // correctly.
-    public static void testOption(String option, String value,
+    public static void testOption(boolean positive,
```

Comment:
Can we rename "positive" to "shouldSucceed" please.

Reply:
I actually started w/ `shouldSucceed`, but it can be kinda misleading as a test succeeds in both cases, it just in one case, it expects JVM to exit normally and in another to hit an error. how about `expectError`?

Reply:
Both "error" and "succeed" have the same problem if you look at it from the perspective of the test rather than the state of the VM being exec'd. How about shouldExitNormally or expectVMFail, or something that makes it VM specific?